src/pegin/components/create/PegInForm.vue

```diff
@@ -153,6 +157,11 @@ export default defineComponent({
       selected.value = selectedType;
     }
 
+    onBeforeMount(() => {
+      const feature = flyoverFeature.value(FeatureNames.FLYOVER_PEG_IN);
+      flyoverEnabled.value = feature?.value === 'enabled';
+    });
+
```

Comment:
Would make sense to add the same for peg-out [here](https://github.com/rsksmart/2wp-app/blob/feature/ff-pegin/src/pegout/views/PegOut.vue)?

Reply:
yes, I'll update it. Thanks